configuration/appframework/scripts/isPatientEnrolledInProgramOfUuid.js

```diff
@@ -0,0 +1,6 @@
+function isPatientEnrolled(programUuid) {
```

Comment:
You need to change this file name to match the function name, now that you have changed it.  Also, this is inconsistent with the way you have implemented the other functions.  The way you have implemented the others is to have all of the arguments you need passed into the function, whereas this one assumes some arguments are already defined as global variables.  We should choose one of these approaches and stick to it for consistency.